### PR TITLE
Return server stats for GetNodeState() interface

### PR DESCRIPTION
**Pull request by alexpan0610**

### CODE REVIEW

File: docs/jsonrpc_apis.md [hunk 1]
```diff
@@ -579,20 +570,19 @@ argument sample:
...
+    "id": 123456,
+    "jsonrpc": "2.0",
+    "result": [
+        "127.0.0.1:64890 (inbound)",
+        "127.0.0.1:64880 (inbound)",
+        "127.0.0.1:64822 (inbound)",
+        "127.0.0.1:22338 (outbound)",
+        "127.0.0.1:23338 (outbound)",
+        "127.0.0.1:24338 (outbound)"
+    ]
```
Comment (bocheng0000) [inline docs/jsonrpc_apis.md]: The result is not matched to the function.

Comment (bocheng0000) [inline docs/jsonrpc_apis.md]: The result is not matched to the function.

Comment (alexpan0610) [reply docs/jsonrpc_apis.md]: The result has changed after merge dpos into dev, so the document needs to update.
image

Comment (bocheng0000) [reply docs/jsonrpc_apis.md]: Please add a description of the result.

Comment (alexpan0610) [reply docs/jsonrpc_apis.md]: The result is just an address in string, no like the GetNodeState() interface the result is struct and have many  parameters.

Comment (bocheng0000) [reply docs/jsonrpc_apis.md]: Maybe you can refer to this document [getneighbors](https://github.com/elastos/Elastos.ELA.SideChain.Token/blob/release_v0.0.1/jsonrpc_apis.md#getneighbors)

File: docs/jsonrpc_apis.md [hunk 2]
```diff
@@ -607,34 +597,33 @@ results:
...
-| Height     | integer | neighbor current height                                         |
-| Services   | integer | neighbor service type. 4 is spv service and 0 is no spv service |
-| Relay      | bool    | whether neighbor will relay transaction or not                  |
-| External   | bool    | whether neighbor is from external network                       |
-| State      | string  | neighbor state in string format                                 |
-| NetAddress | string  | neighbor tcp address                                            |
+| name           | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| ID             | integer | a random number to identify the neighbor instance               |
+| Addr           | string  | network address of the neighbor in host:port format             |
```
Comment (bocheng0000) [inline docs/jsonrpc_apis.md]: Full and lowercase should be used in the interface

Comment (alexpan0610) [reply docs/jsonrpc_apis.md]: Fixed.

File: docs/jsonrpc_apis.md [hunk 3]
```diff
@@ -606,35 +596,33 @@ results:
...
-| HexID      | string  | neighbor's id in hex format                                     |
-| Height     | integer | neighbor current height                                         |
-| Services   | integer | neighbor service type. 4 is spv service and 0 is no spv service |
-| Relay      | bool    | whether neighbor will relay transaction or not                  |
-| External   | bool    | whether neighbor is from external network                       |
-| State      | string  | neighbor state in string format                                 |
-| NetAddress | string  | neighbor tcp address                                            |
+| name           | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| addr           | string  | network address of the neighbor in host:port format             |
```
Comment (bocheng0000) [inline docs/jsonrpc_apis.md]: "address" maybe is better than "addr" as the full name should be used in the interface

Comment (alexpan0610) [reply docs/jsonrpc_apis.md]: Changed to netaddress.

File: docs/jsonrpc_apis.md [hunk 4]
```diff
@@ -606,35 +596,33 @@ results:
...
-| Services   | integer | neighbor service type. 4 is spv service and 0 is no spv service |
-| Relay      | bool    | whether neighbor will relay transaction or not                  |
-| External   | bool    | whether neighbor is from external network                       |
-| State      | string  | neighbor state in string format                                 |
-| NetAddress | string  | neighbor tcp address                                            |
+| name           | type    | description                                                     |
+| -------------- | ------- | --------------------------------------------------------------- |
+| netaddress     | string  | network address of the neighbor in host:port format             |
+| services       | integer | neighbor service type. 4 is spv service and 0 is no spv service |
+| relaytx        | integer | relay transactions to the neighbor or not (0 true/1 false)      |
```
Comment (bocheng0000) [inline docs/jsonrpc_apis.md]: Not matched to the result

Comment (alexpan0610) [reply docs/jsonrpc_apis.md]: This PR is to change the result, you can't see the right result until this PR has merged.

**Pull request merged**